### ISSUE TITLE
go.mod does not comply with semantic versioning

### ISSUE THREAD

**Issue by autodidacticon** [TeaEntityLab/fpGo#2]
```
  go get github.com/TeaEntityLab/fpGo@v2.1.0
go: github.com/TeaEntityLab/fpGo@v2.1.0: invalid version: module contains a go.mod file, so module path must match major version ("github.com/TeaEntityLab/fpGo/v2")

```

Is this project's go.mod correct?

https://github.com/golang/go/issues/35732

**Comment by johnteee** [comment]
https://github.com/TeaEntityLab/fpGo/commit/6be1d563ca4b08606e28f3c2dca642e3cc2a2892 Fixed at v2.1.1

Now the following cmds are valid:

```bash
go get github.com/TeaEntityLab/fpGo/v2
go get github.com/TeaEntityLab/fpGo/v2@v2.1.1
```

**Comment by johnteee** [comment]
Thanks for reporting! 😊
